flang/lib/Lower/Allocatable.cpp

```diff
@@ -204,6 +205,19 @@ class MutablePropertyReader {
     return readBaseAddress();
   }
 
+  /// Return the loaded fir.box.
+  mlir::Value getIrBox() const {
+    assert(irBox);
+    return irBox;
+  }
+
+  /// Read the lower bounds
+  void readLowerBounds(llvm::SmallVectorImpl<mlir::Value> &lbounds) {
```

Comment:
Should this be `getLowerBounds` for consistency? Or maybe `getLBoundsFromOp` if the codegeny aspect is emphasized.

Reply:
Updated as suggested.